### PR TITLE
 feature(docs): badge component theming

### LINKED ISSUE #1039: Theming: Badge
### Subject

Badge

### Description

Create a basic theming documentation for the `Badge` component that showcases how you can style it.

Theming file: [Badge theming file](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/badge/theming.mdx)

### Template Reference

For single part components, feel free to use the `Button` component theming doc as a template reference: [Button theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/button/theming.mdx).

For multipart components, feel free to use the `Input` component theming doc as a template reference: [Input theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/input/theming.mdx).

### PR DESCRIPTION
**Pull request by kelvinwelter**
Closes #1039 

## 📝 Description

Adding theming docs for the `Badge` component

### CODE REVIEW

File: content/docs/components/badge/theming.mdx [hunk 1]
```diff
@@ -3,4 +3,182 @@ id: badge
...
+> [Component Style](/docs/styled-system/component-style#styling-single-part-components)
+> page.
+
+## Theming properties
+
+The properties that affect the theming of the `Badge` component are:
+
+- `variant`: The visual variant of the badge. Defaults to `solid`.
+- `colorScheme`: The color scheme of the badge. Defaults to `gray`.
+- `size`: The size of the badge. Defaults to `md`.
```
Comment (nikolovlazar) [inline content/docs/components/badge/theming.mdx:20]: - The default variant is `subtle` (Check the properties in the [Badge Theme](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/badge.ts))
- The `Badge` component doesn't have a default size.

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
Left you one comment regarding the Theming Properties. Also, there's extra indentation in the whole sandpack code.

File: content/docs/components/badge/theming.mdx (hunk 1 above)
Comment (kelvinwelter) [reply content/docs/components/badge/theming.mdx]: Fixed it on https://github.com/chakra-ui/chakra-ui-docs/pull/1136/commits/dccb6e2c087de07214fb71ad052ea6cf5e93afcd

**Review by nikolovlazar — APPROVED** [review]
There are some minor changes that need to be done, but I'll fix them right away. Thanks for contributing to the Badge theming docs @kelvinwelter 🙌

**Pull request merged**